src/wrp_interface.c

```diff
@@ -85,8 +85,9 @@ ssize_t wi_create_response_to_message(void *data, size_t cnt, void **message)
 	    break;		
     
             default:
-            	return -1;    
-            break;
+            	response.msg_type = WRP_MSG_TYPE__AUTH;                 
+                response.u.auth.status = 501; // Not Implemented
```

Comment:
From WEBPA-1573:- Unless a WRP_MSG_TYPE_REGISTRATION [9] or WRP_MSG_TYPERETRIEVE [6] is received a WRP type WRP_MSG_TYPE_AUTH [2] message with the status "400" shall be returned.